api/views/maps.py

```diff
@@ -0,0 +1,77 @@
+from api import app
+from flask import Blueprint, request
+from api import db
+from api.models import Map
+import json
+from flask import jsonify
+#from api.utils import InvalidUsage
+from sqlalchemy import func
+# import time
+# from datetime import date
+# import uuid
+from flask_sqlalchemy import SQLAlchemy
+from api.utils import create_response
+
+
+
+mod = Blueprint('maps', __name__)
+
+
+#none of these have been comprehsnsively tested yet
+
+@app.route('/maps', methods = ['GET'])
+# Get all map years.
+# get function is definitely not functional
+def get_map_years():
+    if request.method == 'GET':
```

Comment:
since we're splitting each endpoint/request method into their own functions, there's no need to do this check for any of the endpoints